components/map/AvalancheForecastZonePolygon.tsx

```diff
@@ -59,20 +59,18 @@ export const AvalancheForecastZonePolygon: React.FunctionComponent<AvalancheFore
     }
   }, [animationProgress, useAnimation]);
 
-  const polygonProps = {
-    coordinates: toLatLngList(zone.geometry),
+  const polygonProps: GeojsonProps = {
+    geojson: {
+      type: 'FeatureCollection',
+      features: [zone.feature],
+    },
     strokeColor: selected ? highlight.toString() : outline.toString(),
     strokeWidth: selected ? 4 : 2,
     tappable: onPress !== undefined,
     zIndex: selected ? 1 : 0,
-    onPress: (event: PolygonPressEvent) => {
+    onPress: (_event: Parameters<Required<GeojsonProps>['onPress']>[0]) => {
       if (onPress) {
         onPress(zone);
-
-        // By calling stopPropagation, we prevent this event from getting passed to the MapView's onPress handler,
-        // which would then clear the selection
-        // https://github.com/react-native-maps/react-native-maps/issues/1132
-        event.stopPropagation();
```

Comment:
This new kind of event for the `onPress` on `<Geojson>` doesn't have `stopPropagation()` but from my testing it did not seem like we needed it.